### PR TITLE
Upgrade Node version to the latest LTS

### PR DESCRIPTION
**Pull request by barabadzhi**
The issue with Omega invalid headers needs to be retested in case it still persists.
Integrations should be tested as well due to proxy middleware major upgrade.

<del>`DRAFT` until tested.</del>

### CODE REVIEW

File: .github/workflows/deploy-explorer.yml
Comment (nd-novorender) [inline .github/workflows/deploy-explorer.yml:1]: Please try to do formatting in a separate PR next time, harder to see the actual changes

**Review by nd-novorender — COMMENTED** [review]
Some other places where I see node 18:
explorer.bicep:33
nodeWebApp.bicep:33,69

Do you think we should update them as well?

File: .github/workflows/deploy-explorer.yml
Comment (barabadzhi) [reply .github/workflows/deploy-explorer.yml:1]: Sorry, format on save. 🤷‍♂️

**Comment by nd-novorender** [comment]
Looking at https://github.com/novorender/novoweb/commit/c66a7a0addb2d012c739a6ab32d6ab163cd22b40 - there's also https://github.com/novorender/novoweb/blob/feature/upgrade-node-to-latest-lts/infrastructure/modules/nodeWebApp.bicep#L69, by `nodeWebApp.bicep:33,69` I meant lines 33 and 69, my bad :)

**Review by nd-novorender — APPROVED** [review]
